docs/packer.md

```diff
@@ -0,0 +1,32 @@
+# Using Packer and baking AMIs
+
+## Overview
+
+[Packer](http://packer.io/) is well known tool for baking images of any kind.
+We use it to bake our AMIs.
+
+## Prerequisites
+
+* `packer` binary
+* [packer-goss](https://github.com/YaleUniversity/packer-provisioner-goss) plugin
+
+## Plugin instalation
+
+To install `packer-goss` plugin the following should be executed inside of the
+`build/amis/packer` directory:
+
+```bash
+$ curl -o packer-goss  https://github.com/YaleUniversity/packer-provisioner-goss/releases/download/v0.3.0/packer-provisioner-goss-v0.3.0-linux-amd64
```

Comment:
This command as written downloads an html file with a redirect notice rather than the binary itself. Adding the `-L` flag will allow curl to follow the redirect and download the binary. The resulting binary name should also be `packer-provisioner-goss`

```suggestion
$ curl -Lo packer-provisioner-goss  https://github.com/YaleUniversity/packer-provisioner-goss/releases/download/v0.3.0/packer-provisioner-goss-v0.3.0-linux-amd64
```